trackma/engine.py

```diff
@@ -304,6 +304,18 @@ def start(self):
             if os.path.isdir(hooks_dir):
                 import pkgutil
 
+                try:
+                    from importlib.util import module_from_spec
+
+                    def load_hook(loader, name):
+                        spec = loader.find_spec(name)
+                        module = module_from_spec(spec)
+                        spec.loader.exec_module(module)
+                        return module
+                except ImportError:
+                    def load_hook(loader, name):
+                        return loader.find_module(name).load_module(name)
```

Comment:
trackma currently requires Python 3.8, so we don't need this fallback code for <3.5 (which is ancient).

Reply:
I'm stuck in implementing this here... `loader.find_module().load_module()` allowed importing sibling packages, but newer one doesn't... so I'm stuck on this... Is there any way to do this without resorting to sys.path modification?

Reply:
Guess I'll have to take a look at testing this myself. No promises when I'll get to that, though.

Reply:
Found a way to fix sibling imports, but it's not clean...

I should move load_module to utils module...